plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/statistics/DeltaLakeJsonFileStatistics.java

```diff
@@ -141,7 +142,15 @@ private Optional<Object> deserializeStatisticsValue(DeltaLakeColumnHandle column
         if (!columnHandle.isBaseColumn()) {
             return Optional.empty();
         }
-        Object columnValue = deserializeColumnValue(columnHandle, statValue, DeltaLakeJsonFileStatistics::readStatisticsTimestamp, DeltaLakeJsonFileStatistics::readStatisticsTimestampWithZone);
+
+        Object columnValue;
+        try {
+            columnValue = deserializeColumnValue(columnHandle, statValue, DeltaLakeJsonFileStatistics::readStatisticsTimestamp, DeltaLakeJsonFileStatistics::readStatisticsTimestampWithZone);
```

Comment:
Is it hard to add support for the varbinary case ? I would rather we do that than add a try/catch which will hide the underlying problem

Reply:
@raunaqmorarka I'm not entirely sure, and adding the stats here has serious correctness implications (skip a partition that shouldn't be skipped -> incomplete data 😬 ), so I don't feel confident I can come up with a good solution and would rather explicitly skip unsupported statistics in this PR, and let someone else come up with the enhancement in a separate PR.

Additionally, besides VARBINARY, any implementation could come up with statistics for new types, so this PR future-proofs Trino to ensure we'll keep using the stats we know how to handle and avoid the yet-unimplemented ones.

Reply:
@ebyhr @findinpath do you know how to tackle adding support for varbinary stats ? I'd rather have that than swallowing exceptions

Reply:
@raunaqmorarka adding support for varbinary would fix this specific issue, but I still think we should handle unknown types more gracefully, as currently a failure means that _all stats are ignored_, rather than just the unknown stat being ignored. Wdyt?